src/climate_learn/models/hub/linear_regression.py

```diff
@@ -0,0 +1,15 @@
+# Third party
+from torch import nn
+
+
+class LinearRegression(nn.Module):
```

Comment:
I think the classe should be decorated using register (register("linear_regression")).